impossible repair: local variable referenced before assigment

plattform: ocrd/all:maximum
makefile: crop-anyocr-binarize-page-olena-sauvola-denoise-ocropy-deskew-page-ocropy-segment-tesseract-ocropy-dewarp-ocr-ocropy-tesseract (added binarize before anyocr)

# Actual Behavior

Recipe crashing at Segmentation

# Stacktrace

```
07:20:24.307 INFO processor.RepairSegmentation - INPUT FILE 13 / IMG_OCR-D-SEG-BLOCK-tesseract_16457324
Traceback (most recent call last):
  File "/usr/bin/ocrd-segment-repair", line 8, in <module>
    sys.exit(ocrd_segment_repair())
  File "/usr/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/ocrd_segment/cli.py", line 13, in ocrd_segment_repair
    return ocrd_cli_wrap_processor(RepairSegmentation, *args, **kwargs)
  File "/usr/lib/python3.6/site-packages/ocrd/decorators.py", line 54, in ocrd_cli_wrap_processor
    run_processor(processorClass, ocrd_tool, mets, workspace=workspace, **kwargs)
  File "/usr/lib/python3.6/site-packages/ocrd/processor/base.py", line 57, in run_processor
    processor.process()
  File "/usr/lib/python3.6/site-packages/ocrd_segment/repair.py", line 166, in process
    _plausibilize_group(regionspolys, rogroup, mark_for_deletion, mark_for_merging)
  File "/usr/lib/python3.6/site-packages/ocrd_segment/repair.py", line 312, in _plausibilize_group
    for elem in regionrefs:
UnboundLocalError: local variable 'regionrefs' referenced before assignment
Makefile:304: recipe for target 'OCR-D-plausible' failed
make[1]: *** [OCR-D-plausible] Error 1
make[1]: Leaving directory '/data'
Makefile:194: recipe for target '.' failed
make: Leaving directory '/data'
make: *** [.] Error 2
```

## Suggested Fix

Add `regionrefs = list()` to the initalized vars, like this: 
```
def _plausibilize_group(regionspolys, rogroup, mark_for_deletion, mark_for_merging):
    wait_for_deletion = list()
    reading_order = dict()
    regionrefs = list()
```





```

--- a/ocrd_segment/repair.py
+++ b/ocrd_segment/repair.py
@@ -316,6 +316,7 @@ def _child_within_parent(child, parent):
 def _plausibilize_group(regionspolys, rogroup, mark_for_deletion, mark_for_merging):
     wait_for_deletion = list()
     reading_order = dict()
+    regionrefs = list()
     ordered = False
     if isinstance(rogroup, (OrderedGroupType, OrderedGroupIndexedType)):
         regionrefs = (rogroup.get_RegionRefIndexed() +

